core/modules/wsched/BlendScheduler.h

```diff
@@ -62,6 +62,8 @@ class BlendScheduler : public wcontrol::Scheduler {
     void commandStart(util::Command::Ptr const& cmd) override;
     void commandFinish(util::Command::Ptr const& cmd) override;
 
+    bool ready();
```

Comment:
It's not really "const", aside from locking a mutex, the call to _scan->ready() may manipulate its data structures.